plugin/src/main/java/io/jenkins/plugins/casc/Attribute.java

```diff
@@ -111,6 +112,10 @@ boolean isIgnored() {
         return restrictions != null ? restrictions : EMPTY;
     }
 
+    public void setJsonSchema(boolean jsonSchema) {
```

Comment:
Add Javadoc?

Reply:
Added  +1

Reply:
is this still required?

Reply:
Sorry for closing now it is the separate thread